api/app/db/crud/hfi_calc.py

```diff
@@ -25,8 +25,20 @@ def get_all_stations(session: Session) -> CursorResult:
     return session.query(PlanningWeatherStation.station_code).all()
 
 
+def get_fire_centre_station_codes() -> List[int]:
```

Comment:
This used to be in utils - but I think it belongs right here in the crud layer. Asking for a list of station codes from the database.